src/components/scenes/Fio/FioCreateHandleScene.tsx

```diff
@@ -71,6 +71,9 @@ export const FioCreateHandleScene = ({ navigation, route }: Props) => {
 
   // TODO: Give feedback to the user to indicate why the input was rejected
   const handleChangeFioHandle = useHandler((userInput: string) => {
+    // Clean spaces:
+    userInput = userInput.replace(domainStr, '').trim()
+
```

Comment:
why did this get added back. There should be no domain in the string

Reply:
You're right, intended to ensure spaces were cleaned too but it's already handled.